extension/package.json

```diff
@@ -125,7 +125,7 @@
       },
       {
         "command": "gradle.runBuild",
-        "title": "Run a Gradle Build",
+        "title": "Gradle: Run a Gradle Build",
```

Comment:
try below?
```
group: Gradle
title: something
```

Reply:
good suggestion, and here we can use `category`.